chrome/content/utils/accounts.js

```diff
@@ -74,7 +74,7 @@ var gmanager_Accounts = new function() {
   this.openFolder = function(aEmail, aLocation, aFolderId) {
     if (this._manager.isAccount(aEmail)) {
       var account = this._manager.getAccount(aEmail);
-      var folder = account.getFolder(null, aFolderId);
+      var folder = account.getFolder(account.password, aFolderId);
```

Comment:
same 😋 
and see lines 102-104 with similar structure `account.getCompose(null, aHref)` (all 3 called with `null`)